Decentralization - Using Pledge as a Bidding Parameter to Determine Saturation Limit.md

```diff
@@ -0,0 +1,233 @@
+**_α<sub>unc</sub>_**  = saturation limit left unclaimed even after some pools have over-pledged. This variable is defined in Equation 7.\
+**_T<sub>a</sub>_** = pool accumulated time of operation (in days) counting from day 1 of operation (read explanation below).\
+**_T<sub>tot</sub>_**= total accumulated time of operation (in days) of all pools (read explanation below).
+
+The first term of Equation 2 allocates saturation limit based on pledge, and ensures that saturation limit increases or decreases exponentially with pledge.\
+\
+ It is ideal that **_α<sub>unc</sub>_** = 0, but if **_α<sub>unc</sub>_** > 0, the second term of Equation 2 ensures its distribution. The distributed **_α<sub>unc</sub>_**, however, can still be re-claimed by under-pledged pools by increasing their pledges.\
+ \
+Distribution of **_α<sub>unc</sub>_** based, yet again, on pledge will further disadvantage those who are under-pledged. Therefore, we use accumulated time of operation,**_T<sub>a</sub>_**, as a parameter for the distribution of **_α<sub>unc</sub>_** which, in turn, ensures that the unclaimed saturation limit is distributed into pools that had been securing the ecosystem for the longest time. This manner of distribution prevents any would-be attacker, likely someone who had just registered a pool (or pools), from getting a significant portion of **_α<sub>unc</sub>_**. This mechanism is very important when the number of pools (**_k_**) suddenly decreased to a very low number, making **_p<sub>opt</sub>_** very expensive and leading to significant under-pledging.
+
+\
+**Example 1**
+* **_k_** = 3119 (epoch 321)
+* **_p<sub>opt</sub>_** = 47,205 ADA (at **_n_** = 15, see calculation in the previous section)
+* **_p<sub>a</sub>_** = 47,205 ADA
+* **_α<sub>unc</sub>_**  = 0 (we will deal with nonzero values later)
+* **_φ_** = 33B (current ADA in circulation)\
+  \
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; **_α<sub>unc</sub>_** = $\frac{33B}{3119}$ $e^{1 - 47,205/47,205}$ = $\frac{33B}{3119}$ = 10.6M\
+\
+**Conclusion**: Epoch 321 saturation limit for pools with at least 47,205 ADA in pledge is 10.6M (at **_n_** = 15). The ratio $\frac{33B}{3119}$ guarantees that every ADA in circulation has a pool to stake to without risking oversaturation. Note that the current average stake per pool is only 7.6M, suggesting that a vast number of pools are not adequately staked. Setting the saturation limit, on average, to 10.6M would make some large pools to become oversaturated which will encourage delegators of these pools to move their stakes to small pools.\
+&nbsp;
+
+**Example 2** (same variable values as in example 1 but lower **_p<sub>a</sub>_**).
+* **_k_** = 3119 (epoch 321)
+* **_p<sub>opt</sub>_** = 47,205 ADA
+* **_p<sub>a</sub>_** = 30,000 ADA (for example)
+* **_α<sub>unc</sub>_**  = 0 (we will deal with nonzero values later)
+* **_φ_** = 33B (current ADA in circulation)\
+  \
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; **_α_** = $\frac{33B}{3119}$ $e^{1 - 47,205/30000}$ = 6.0 M\

```

Comment:
Rely on `&nbsp;` to create fixed amounts of white space is non-portable, and the consequences of those raw characters showing up in CIP text would be worse than not being able to fine-tune the white space.  Leaving everything (both text & equations) left-justifed would be universally readable.